doc/release-notes-bucash.md

```diff
@@ -12,11 +12,19 @@ Please report bugs using the issue tracker at github:
 This is a major release version based of Bitcoin Unlimited compatible
 with the Bitcoin Cash specifications you could find here:
 
-
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/uahf-technical-spec.md (Aug 1st '17 Protocol Upgrade, bucash 1.1.0.0)
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17 Protocol Upgrade, bucash 1.1.2.0)
 - https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-hardfork.md (May 15th '18 Protocol Upgrade, bucash 1.3.0.0, 1.3.0.1, 1.4.0.0)
-- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0, 1.5.0.1)
+
+This release will also implements also a set of consensus changes proposed by an alternative implementation, Bitcoin SV,
```

Comment:
-This release will also implements also a
+This release will also implement a

Reply:
thanks will fix